src/OpenLoco/Windows/StationWindow.cpp

```diff
@@ -847,9 +847,6 @@ namespace OpenLoco::Ui::Windows::Station
             if (callingWidget != Common::widx::caption)
                 return;
 
-            if (strlen(input) == 0)
```

Comment:
Is it safe to do the game command with an empty string? Guess its just outputting memory that is ignored. Perhaps we should add a comment.

Reply:
If the input string is empty, a name will be generated instead.